src/features/analytics/events.ts

```diff
@@ -0,0 +1,97 @@
+export enum Events {
+  PageView = '$pageview',
+
+  // create wallet
+  OnboardingCreateClick = 'onboarding | new wallet | create | click',
+  OnboardingCreateWritePassphraseNextClick = 'onboarding | new wallet | write passphrase | next | click',
+  OnboardingCreateEnterPassphraseNextClick = 'onboarding | new wallet | enter passphrase | next | click',
+  OnboardingCreateEnterPassphraseSkipClick = 'onboarding | new wallet | enter passphrase | skip | click',
+  OnboardingCreateWalletNamePasswordNextClick = 'onboarding | new wallet | wallet name & password | next | click',
+
+  // import wallet
+  OnboardingRestoreClick = 'onboarding | restore wallet | click',
+  OnboardingRestoreEnterPassphraseNextClick = 'onboarding | restore wallet | enter passphrase | next | click',
+  OnboardingRestoreWalletNamePasswordNextClick = 'onboarding | restore wallet | wallet name & password | next | click',
+
+  // receive
+  ReceiveClick = 'receive | receive | click',
+  ReceiveCopyAddressIconClick = 'receive | receive | copy address icon | click',
+
+  // send
+  SendClick = 'send | send | click',
+  SendTransactionDataReviewTransactionClick = 'send | transaction data | review transaction | click',
+  SendTransactionConfirmationConfirmClick = 'send | transaction confirmation | confirm | click',
+  SendTransactionConfirmed = 'send | transaction confirmed',
+
+  // settings
+  SettingsNetworkPreviewClick = 'settings | network | preview | click',
+  SettingsNetworkPreprodClick = 'settings | network | preprod | click',
+  SettingsNetworkMainnetClick = 'settings | network | mainnet | click',
+  SettingsNetworkTestnetClick = 'settings | network | testnet | click',
```

Comment:
Did you end up leaving testnet in place @renanvalentin ?

Reply:
Remove it here: [b16443a7930370ba7fe9541766b8255b326c3009](https://github.com/input-output-hk/nami/pull/893/commits/0cca1097adb05f332fa63fd0b6f981319a72d111)